packages/orbit-components/src/Modal/index.js.flow

```diff
@@ -37,6 +37,7 @@ export type Props = {|
   +isMobileFullPage?: boolean,
   +preventOverlayClose?: boolean,
   +hasCloseButton?: boolean,
+  +disableAnimation?: boolean,
```

Comment:
Don't forget to add it also into `index.d.ts` file 🙏

Reply:
It exists already. 🤔